egs2/mini_an4/spk1/local/make_trial.py

```diff
@@ -0,0 +1,14 @@
+import sys, os
+
+if __name__ == "__main__":
+    with open(sys.argv[1]) as f:
+        lines = f.readlines()
+
+    joint_key = lines[0].strip().split(" ")[0] + "*" + lines[1].strip().split(" ")[0]
+    with open(os.path.join(sys.argv[2], "trial.scp"), "w") as f:
```

Comment:
Maybe it's better to rename `trial.scp` to `trial1.scp` to make them more consistent.